docs/getting_started/dashboard/dashboard_block_reference.md

```diff
@@ -0,0 +1,170 @@
+
+- **Name** - Enter a name for the block.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of content and submit your changes.
+
+## Ibexa News block
+
+Presents a list of recent blog posts or articles published at `ibexa.co` blog (product related).
+It includes title, image, publication date, and link to article details.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **News limit** - Set a maximum number of news to be displayed. Default = 7, minimum value = 1, and maximum = 10.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of news and submit your changes.
+
+## My content block
+
+Displays a list of content items created by the user who is currently logged in.
+It shows the following tabs: **Drafts**, **Scheduled**, **Content**, **Media**, **Drafts to review**.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of content and submit your changes.
+
+## Orders by status block
+
+Displays a chart presenting orders split by status, with their number and percentage.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Statuses** - Set the statuses of orders that should be included in the list: Pending, Processing, Cancelled, Completed. Default value = All.
+- **Periods** - Set the time period: All time, Last day, Last week, Last month. Default value = Last month.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of orders and submit your changes.
+
+## Products by category block
+
+Displays a chart presenting the products split by category, together with the percentage.
+Some products can be assigned to multiple categories, or be uncategorized.
+You can choose to show or hide uncategorized products in the chart.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Categories limit** - Set a maximum number of categories to be displayed.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
+
+## Products with lowest stock block
+
+Displays a table of products with the lowest stock.
+Products are sorted based on Stock value (sorted from lowest to highest stock).
+
+The table contains the following columns: Name, Image, Code, Category, Type, Variant, Stock.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Number of products to be displayed** - Set a number of products to be displayed. Default value = 10.
+- **Stock** - Enter the number of products in stock. Default value = 10.
```

Comment:
We should explain it in different way: set up the maximum stock value (only product with stock >=0 and <=X will be displayed)